test/scripts/watch-replication

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
I am not sure this belongs with this PR. We start with a utility and then integrate it with the flow, at which point it becomes difficult to extricate it with the overall scheme of testing and samples. Merging it for now, let's see what the future brings.

Reply:
This is just a tool I added to demonstrate how to wait until changes are replicated during the demo. With openshift console we can use the failover dialog showing the last sync time.

I think something providing this functionality can be useful in the kubectl-ramen plugin in the future.